### PR TITLE
feat(components): add progressbar component

### PR DESCRIPTION
**Pull request by apvale**
## Description of the change

This PR adds the first version of ProgressBar component. 

image

## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: packages/components/src/components/ProgressBar/ProgressBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { ProgressBar } from "./ProgressBar";
+
+describe("<ProgressBar />", () => {
+  it("renders the default correctly with a value", () => {
```
Comment (nyan07) [inline packages/components/src/components/ProgressBar/ProgressBar.test.tsx:6]:
```suggestion
  it("renders the default variant correctly", () => {
```

File: packages/components/src/components/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    props.color === "green"
+      ? theme.colors.colorIconSuccess
+      : theme.colors.colorBackgroundPrimaryWeak};
+`;
+
+export interface ProgressBarProps extends Omit<ProgressProps, "value"> {
+  /** The progress value. */
+  value: number;
+  /** The color of the progress bar. */
+  color?: "blue" | "green";
```
Comment (nyan07) [inline packages/components/src/components/ProgressBar/ProgressBar.tsx:28]: Should the color be controlled inside the component? I understood that the bar turns green when it is 100%.

File: packages/components/package.json [hunk 3]
```diff
@@ -69,6 +69,7 @@
     "extends @localyze-pluto/browserslist-config"
   ],
   "dependencies": {
+    "@radix-ui/react-progress": "^1.0.1",
```
Comment (nyan07) [inline packages/components/package.json:72]: Just wondering if it should be a dependency or a peerDependency :thinking:

File: packages/components/src/components/ProgressBar/ProgressBar.tsx (hunk 2 above)
Comment (apvale) [reply packages/components/src/components/ProgressBar/ProgressBar.tsx]: We have some [additional design on Figma](https://www.figma.com/file/0KtV7fzjug9hL3XOLzqn1l/%5B-WIP-%5D-%F0%9F%AA%90-Pluto-Components?node-id=387%3A2956) that shows the 100% as blue. So, I decided to remove the prop color for this first version and add it if necessary soon.

File: packages/components/package.json (hunk 3 above)
Comment (apvale) [reply packages/components/package.json]: Moved to peerDependency.

**Review by richbachman — APPROVED** [review]
:shipit:

**Review by nyan07 — APPROVED** [review]

**Pull request merged**